### DISCUSSION TITLE
Thanks for this software!!

### DISCUSSION THREAD

**Discussion by clarkerz** [jekhokie/raspberry-noaa-v2#273] [General]
I've got this working on a Raspberry PI 2B with no issues at all - it said to mention if you've got it working on a different RPI. The only thing I had to do was delay the schedule script running for 60 seconds after reboot as even though the output.log said it had scheduled, when you run an atq command there was nothing scheduled. Adding a 60 second delay to run the script after reboot fixed this. 

Could this be migrated to a i5/amd64 ubuntu system, or is that not possible/too much hassle?? 

Thanks for providing this software, really appreciated. :)

**Comment by jekhokie** [comment]
@clarkerz very cool thanks for the info, I'll update the supported hardware!

The migration to ubuntu is a bit of a lift. There have been a couple attempts but we don't have a ton of people yet who are really wanting this, so we've been focusing our efforts on the RasPi version of this, apologies!

**Reply by clarkerz** [reply]
Please don't appologise Justin...it was just a thought.  I have a fairly decent Ubuntu server running as a DAB Modulator, Mux and encoder  and also an Icecast2 server and only using about 5% of resources - but I am VERY happy with the PI. :)

Is there an easy way to delete a scheduled pass in the event of a clash and a better satellite being classed as a conflict? My next challenge (obession) is to play with the gain as I currently have it set to auto. I am using a NESDR SMArTee v2 with a NooElec SAWbird+ NOAA Filter & Cascaded LNA at the 3d printed 137Mhz v-dipole aerial. I am getting some great pictures with passes I would never have bothered with previously, using my FT- 818 due to the elevation not being directly overhead. Have a bit of banding to address.  
Last question (I promise), do you need to run the install and upgrade script each time you change any settings in the setting.yml file. i.e gain etc?

Thanks again for your all hard work with this project! PS. The PI I am using is a  Raspberry Pi 2 Model B Rev 1.1.

Andy

**Reply by jekhokie** [reply]
There is a feature request in the queue for both advanced scheduling where the user can select a method for prioritization (first available, higher elevation, etc.) and also a feature to enable webpanel-based overrides for scheduling, yes. Don't know when they will be completed, but they're in the queue.  :-)

And yes - any time you update the `config/settings.yml` file you need to run the `./install_and_upgrade.sh` script - I developed that script to be fully idempotent to allow an easy "run whenever/however frequent the user wants" way to keep things up to date and upgrade without having to check multiple different places for propagating the configurations.

**Reply by clarkerz** [reply]
That's great! Thanks Justin. 

Andy
